server/helpers/cleanupArticlesResponse.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @description function to return well formatted articles
+ * @param {array} rows
+   * @returns {object} returns articles object
+*/
+const cleanupArticlesResponse = (rows) => {
+  const result = [];
+  for (let x = 0; x < rows.length; x += 1) {
```

Comment:
Good job. Since you installed the `object-spread-destructuring` babel plugin. Consider drying up this function into something very brief.
An example could be:
```
    const { author, category } = rows[x].dataValues;
    const newObj = {
      ...rows[x].dataValues,
      category: category.categoryName,
      author
    };
    result.push(newObj);
```

Reply:
@augustineezinwa Good suggestion. Except I only need certain things from the rows[x].dataValues Object and not everything within it. If I spread it, I get everything that I do not need in the result, which defeats the purpose of this function.

Reply:
You can eliminate all unneeded object keys by setting them to undefined.
example:
```
rows[x].dataValues[unneededObjectKey] = undefined;

```